packages/styles/scss/components/lightbox-media-viewer/_lightbox-media-viewer.scss

```diff
@@ -34,6 +40,10 @@
         color: $text-02;
       }
     }
+    &__videoPlayer {
+      width: '400px';
+      height: '400px';
+    }
```

Comment:
Let's not include any coding for the embed for this story.

Reply:
sorry for testing purpose I have added these, will remove them in a final PR.